services/core-api/app/api/now_applications/models/now_application_progress.py

```diff
@@ -55,12 +56,3 @@ def create(cls, now_application, application_progress_status_code, add_to_sessio
     @classmethod
     def get_active(cls):
         return cls.query.filter_by(active_ind=True).all()
-
-    @validates('end_date')
-    def validate_end_date(self, key, end_date):
-        if end_date is not None:
-            if isinstance(end_date, str):
-                end_date = dateutil.parser.isoparse(end_date)
-            if end_date < self.start_date:
-                raise AssertionError('end_date cannot be before start_date')
-        return end_date
```

Comment:
Moved validations to the FE as 'start_date' needs to be compared against the new value not the db saved value